src/Components/Homepage.jsx

```diff
@@ -0,0 +1,28 @@
+import '../index.css';
+import { NavLink } from 'react-router-dom';
+
+const Homepage = () => (
+  <div className="bgcontainer">
+    <div className="getposition">
+      <div className="flex justify-center  items-center">
+        <h1 className="text-3xl text-center text-black font-bold">Welcome to the E-Bike Shop</h1>
+      </div>
+      <div className="flex justify-center items-center">
+        <ul className="flex gap-4 mt-5">
+          <li className="btnone">
+            <NavLink to="/login">
+              <h2>Login</h2>
+            </NavLink>
+          </li>
+          <li className="btnone">
+            <NavLink to="/signup">
+              <h2>Sign Up</h2>
+            </NavLink>
```

Comment:
Good job @tchasinga  👍🏼  one small thing to mention here is the design and the colors , in the project requirements we were asked to stick to the design in terms of layout colors and fonts  

The result should look exactly the same (with exception for titles and images) as in [the design](https://www.behance.net/gallery/26425031/Vespa-Responsive-Redesign) provided by [Murat Korkmaz](https://www.behance.net/muratk) on Behance.